PKGBUILD

```diff
@@ -38,5 +44,8 @@ package() {
 
   cd "${srcdir}/${pkgname}-${pkgver}"
   python setup.py install --prefix=/usr --root="${pkgdir}" --optimize=1
+  
   install -D -m644 COPYING "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+  install -D -m644 "${srcdir}/${pkgname}.desktop" "${pkgdir}/usr/share/applications/${pkgname}.desktop"
+  install -D -m644 "${srcdir}/${pkgname}-${pkgver}/data/${pkgname}_halo.svg" "${pkgdir}/usr/share/pixmaps/${pkgname}.svg"
```

Comment:
Shouldn't the icon go into `/usr/share/icons/hicolor/scalable/apps/`? We could also add some of the raster versions (which are all based on the halo variant):
```sh
for size in 16 24 32 48 64 256; do
  install -D -m644 "data/paperwork_${size}.png" \
    "${pkgdir}/usr/share/icons/hicolor/${size}x${size}/apps/paperwork.png"
done
```